stdlib/public/SwiftShims/CFHashingShims.h

```diff
@@ -0,0 +1,41 @@
+//===--- CFHashingShims.h - CoreFoundation declarations for CF hashing functions ------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+CF_IMPLICIT_BRIDGING_ENABLED
+CF_EXTERN_C_BEGIN
+_Pragma("clang assume_nonnull begin")
+
+
+#define _CF_HASHFACTOR 2654435761U
+
+CF_INLINE CFHashCode __CFHashInt(long i) {
```

Comment:
Because they are re-exports of CF internals. I don't want to maintain two implementations in two different languages and worry about casting differences etc. Much easier to fix bugs once and copy paste the fix (would be better to not have to copy paste.....)